types/address/hash_test.go

```diff
@@ -0,0 +1,90 @@
+package address
+
+import (
+	"crypto/sha256"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestAddressSuite(t *testing.T) {
+	suite.Run(t, new(AddressSuite))
+}
+
+type AddressSuite struct{ suite.Suite }
+
+func (suite *AddressSuite) TestHash() {
```

Comment:
Mind replace `suite` with something else? It clashes with the `github.com/stretchr/testify/suite` import

```suggestion
func (addressSuite *AddressSuite) TestHash() {
```

Reply:
this makes the names longer, and in fact we don't need `suite` package inside the function. In the package examples: https://github.com/stretchr/testify#suite-package they also use `suite` for self reference. If you are not convinced, then how about, `as`?

Reply:
Isnt it typical to use one letter identifiers for the target of go methods? Just `s`?

I am often confused why the cosmos sdk uses idioms different than most major go repos.

Reply:
Yes, I was already advocating in many places for short yet meaningful names. I think `suite` is the most widely used, but I can change to `s` if nobody is against.